aries-site/src/examples/components/tag/CreateTag/TagPageHeader.js

```diff
@@ -1,22 +1,27 @@
+import { useContext } from 'react';
 import PropTypes from 'prop-types';
 import { Box, Heading, Text } from 'grommet';
+import { PageContainerContext } from '../../..';
 
-export const TagPageHeader = ({ description }) => (
-  <Box gap="medium" flex={false}>
-    <Box gap="xsmall" flex={false}>
-      <Heading level={2} margin="none">
-        Create and Assign Tags
-      </Heading>
+export const TagPageHeader = ({ description }) => {
+  const { ...pageContainer } = useContext(PageContainerContext);
```

Comment:
I this change related? I'm not immediately seeing the connection.

Reply:
Thanks for catching. It is not relevant. I will remove.